doc: ble_controller: Document known toolchain issue (#46)
NordicPlayground/nrfxlib: ble_controller/README.rst

  * The libraries are for evaluation purposes only.
  * The libraries are not fully functional.
  * The libraries are not built for performance.

>   * Certain toolchains are known to segfault when linking with the libraries.

Reviewer: ```suggestion
  * Some toolchains cause a segmentation fault when linking the libraries.
```

Reviewer: Not sure about "linking the libraries" - "linking in the libraries" or "linking to the libraries" might be better, depending on what you want to say.

Author: good catch, "linking with the libraries" does indeed communicate the wrong thing.
